src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/HistoricalDataConstants.java

```diff
@@ -55,4 +55,10 @@
 
   /** Minimum amount of Historical FIFO queue that can be set by library users. */
   static final long MIN_QUEUE_SPAN_MINS = 1;
+
+  /** The number of milliseconds in one hour. */
+  static final long MILLISECONDS_PER_HOUR = 3600000L;
+
+  /** Default maximum amount of time Historical FIFO can get behind. */
+  static final long MAX_HIST_FIFO_GET_BEHIND_MS = 48L * MILLISECONDS_PER_HOUR;
```

Comment:
See comment at https://github.com/hms-networks/sc-ewon-flexy-extensions-lib/pull/16#discussion_r799516012

Reply:
oh, I see what you mean.

Reply:
changed